src/scope-for-gradle.ts

```diff
@@ -2,18 +2,15 @@ import * as core from '@actions/core'
 import * as executor from './executor'
```

Comment:
this usually makes easier code:
```suggestion
import { instrument } from './executor'
```